content/docs/command-reference/metrics/index.md

```diff
@@ -54,7 +103,7 @@ and manage <abbr>DVC project</abbr> metrics.
 
 ## Examples
 
-First, let's create a simple DVC-file:
+First, let's create a simple DVC pipeline stage:
```

Comment:
```suggestion
First, let's create a simple stage:
```
Maybe link "stage" to the run cmd ref, if no other links to that doc exist nearby.